fv3gfs-physics/fv3gfs/physics/stencils/microphysics.py

```diff
@@ -11,6 +11,8 @@
 from pace.dsl.typing import Float, FloatField, FloatFieldIJ, Int
 from pace.util.grid import GridData
 
+from .._config import PhysicsConfig
```

Comment:
I think you put it in the top level too right? Can we do `from fv3gfs.physics import PhysicsConfig`? The two dots are messing with my brain

Reply:
The short version is, within a package we need to use relative imports - top level imports are for "other" packages (e.g. in physics, for stencils/dsl/util).

Within the `physics` package, we can't import everything from `physics` top level, because this would cause a variety of circular dependencies. In this case, `physics.__init__` imports `physics.stencils.microphysics.Microphysics` which would in turn import `physics.__init__` causing a circular dependency.